### DISCUSSION TITLE
help with the launch

### DISCUSSION THREAD

**Discussion by FdkIdk** [IMXNOOBX/cs2-external-esp#187] [Q&A]
Greetings, I am faced with the problem of launching this esp. Esp is not displayed at all, although offsets are updated (attached a screenshot). There are a couple of questions, since I'm not really experienced in this. 
1. How to launch esp correctly? through 
 .exe or via a project .sln?
I tried both options, it doesn't work. 
2. Maybe someone has encountered a problem like mine? 
![1212](https://github.com/user-attachments/assets/23367dee-0d97-4a2d-979c-02f43f2a3849)

**Comment by IMXNOOBX** [comment]
Im sorry for the late response, i don't get notifications for this posts. You should run the exe file, and try running as administrator
